src/System.Windows.Forms.Primitives/src/Interop/Ole32/Interop.DEC.cs

```diff
@@ -0,0 +1,18 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+internal static partial class Interop
+{
+    internal static partial class Ole32
+    {
+        public struct DEC
+        {
+            public uint wReserved;
```

Comment:
In the C++ headers this is USHORT, using uint is wrong

```suggestion
            public ushort wReserved;
```